src/helper/helper.js

```diff
@@ -298,6 +313,17 @@ helper.isArray_ = function(value) {
   return plain.type(value) == 'array';
 };
 
+/**
+ * Determines if the given value is an arguments object.
+ *
+ * @param {*} value The value to test.
+ * @return {boolean} True iff the given value is an arguments object.
+ * @private
+ */
+helper.isArguments_ = function(value) {
+  return plain.type(value) == 'arguments';
```

Comment:
Style nit: Use triple-equals except for null and undefined:
https://google.github.io/styleguide/jsguide.html#features-equality-checks

(No need to update preexisting comparisons in this PR).